### PR TITLE
Add install func

### PR DESCRIPTION
**Pull request by jesusbv**

### CODE REVIEW

**Comment by jesusbv** [comment]
This PR is based on #2 , PR #2 should be merged first

File: main/ahb_for_sles.go [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+  }
+  return hasPubCloud, version, nil
+}
+
+func getAhbInfo() (AHBInfo) {
+  return AHBInfo{
+    PublicCloudService: "public_cloud",
+    RegisterCloudGuestPath: "/usr/sbin/registercloudguest",
+    RegionSrvMinVer: "9.3.1",
+    RegionSrvEnablerService: "regionsrv-enabler-azure.service",
```
Comment (rjschwei) [inline main/ahb_for_sles.go:125]: use the timer s/service/timer/

File: main/ahb_for_sles.go [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+    RegionSrv: "cloud-regionsrv-client",
+    RegionSrvAddOn: "cloud-regionsrv-client-addon-azure",
+    AddonPath: "/usrb/sbin/regionsrv-enabler-azure",
+    RepoAlias: "sle-ahb-packages",
+    ModName: "sle-module-public-cloud",
+    RepoUrl: "https://updates.suse.com/SUSE/Updates/SLE-Module-Public-Cloud-Unrestricted/%s/x86_64/update",
+  }
```
Comment (rjschwei) [inline main/ahb_for_sles.go:132]: We need to track the additional packages that may need installation if they are not there:

cloud-regionsrv-client-plugin-azure
regionServiceClientConfigAzure
regionServiceCertsAzure

When a customer brings their own image and wants to turn it into a PAYG instance it is unlikely they will have those packages.

**Review by rjschwei — CHANGES_REQUESTED** [review]

File: main/ahb_for_sles.go (hunk 2 above)
Comment (jesusbv) [reply main/ahb_for_sles.go:295]: Done

**Review by rjschwei — APPROVED** [review]

**Pull request merged**